decentraland/unity-explorer
# fix: avoid showing old chat bubbles when re-enabling nametags

## What does this PR change?

Fix #2257
This PR fixes the issue that caused chat bubbles to appear with old messages on top of users

...

## How to test the changes?



1. Launch the explorer
2. disable nametags
3. write in the chat a few messages
4. re-enable nametags quickly
5. verify no chat bubbles with old messages appear in your nametag

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
